header.scss

```diff
@@ -0,0 +1,397 @@
+              -webkit-transform: rotate(-45deg)
+            }
+
+            &::after {
+              top: 50%;
+              transform: rotate(45deg) translate(0px 50%);
+              -webkit-transform: rotate(45deg)
+            }
+          }
+        }
+
+
+        nav {
+          position: fixed;
+          top: 0;
+          left: -100%;
+          width: 100%;
+          height: 100%;
+          background-color: $color-primary;
+          padding-top: 92px;
+          transition: left 0.3s ease 0s;
+          overflow: auto;
+          z-index: 4;
+
+          ul {
+            li {
+              list-style-type: none;
+              display: flex;
+              flex-direction: column;
+              align-items: center;
+
+              a {
+                text-decoration: none;
+                text-transform: uppercase;
+                @include fontProp(12px, 24px, $color-link);
+                letter-spacing: 2px;
+                padding: 8px;
+
+                &:hover {
+                  @include fontProp(14px, 24px, $color-link-hover);
+                }
+
+                &.current {
+                  padding-bottom: 3px;
+                  display: flex;
+                  flex-direction: column;
+                  align-items: center;
+
+                  &:after {
+                    content: "";
+                    width: 40px;
+                    height: 2px;
+                    background-color: $color-link;
+                    display: block;
+                    text-align: center;
+                    transition: all 0.3s ease 0s;
+                  }
+                }
+              }
+
+
+              &.get-skout-btn, &.login-btn {
+                button {
+                  margin: 197px 0 16px 0;
+                  border: 1px solid $color-link;
+                  border-radius: 20px;
+                  width: 180px;
+                  text-align: center;
+                  @include fontProp(12px, 24px, $color-link);
+                  letter-spacing: 2px;
+                  padding: 8px;
+                  text-transform: uppercase;
+                  background-color: $color-primary;
+                  font-family: 'Acme', sans-serif;
+
+                  &:hover {
+                    @include fontProp(14px, 24px, $color-link-hover);
+                    color: $color-primary;
+                    background-color: $color-link;
+                  }
+                }
+              }
+
+              &.login-btn {
+                button {
+                  width: 90px;
+                  margin: 0 0 40px;
+                }
+              }
+            }
+          }
+
+          &.active {
+            left: 0;
+
+            &::before {
+              content: "";
+              position: fixed;
+              width: 100%;
+              top: 0;
+              left: 0;
+              height: 62px;
+              background: linear-gradient(45deg, #12C1E6, #01F973);
```

Comment:
duplicate code in line 9, you can use var for gradient

Reply:
done

Reply:
It`s not a duplicate code. It`s  code for different blocks.